sdk/template/.template.config/template.json

```diff
@@ -438,6 +436,14 @@
           }
         ]
       }
+    },
+    "ciFileName": {
+      "type": "parameter",
+      "datatype":"text",
+      "isRequired": false,
+      "defaultValue": "ci",
+      "description": "ci file name.",
+      "fileRename": "ci-template"
```

Comment:
How does this map to the file? Does it ignore the file extension?

Reply:
No, it is just replace 'çi-template' to 'ci' for the file name.
We have changed to use 'rename directive' to do this.